From c9652a164a456c6d0c4e1ec21d1dd84606d534a1 Mon Sep 17 00:00:00 2001
From: Stephane Desneux <stephane.desneux@open.eurogiciel.org>
Date: Mon, 29 Sep 2014 19:19:31 +0200
Subject: [PATCH 13/24] 55-udev-smack-default.rules updated for systemd 212

SMACK="<label>" *MUST* be changed to SECLABEL{smack}="<label>" in udev rules

Change-Id: Ib1a2f70e9b3e20a71cb4b4e05a2c03e574af8c2d
Bug-Tizen: TC-660
Signed-off-by: Stephane Desneux <stephane.desneux@open.eurogiciel.org>
---
 rules/55-udev-smack-default.rules | 36 ++++++++++++++++++------------------
 1 file changed, 18 insertions(+), 18 deletions(-)

diff --git a/rules/55-udev-smack-default.rules b/rules/55-udev-smack-default.rules
index dd45c51..3829019 100644
--- a/rules/55-udev-smack-default.rules
+++ b/rules/55-udev-smack-default.rules
@@ -1,23 +1,23 @@
 # do not edit this file, it will be overwritten on update
 
-KERNEL=="null", SMACK="*"
-KERNEL=="zero", SMACK="*"
-KERNEL=="console", SMACK="*"
-KERNEL=="kmsg", SMACK="*"
-KERNEL=="video*", SMACK="*"
-KERNEL=="card*", SMACK="*"
-KERNEL=="ptmx", SMACK="*"
-KERNEL=="tty", SMACK="*"
+KERNEL=="null", SECLABEL{smack}="*"
+KERNEL=="zero", SECLABEL{smack}="*"
+KERNEL=="console", SECLABEL{smack}="*"
+KERNEL=="kmsg", SECLABEL{smack}="*"
+KERNEL=="video*", SECLABEL{smack}="*"
+KERNEL=="card*", SECLABEL{smack}="*"
+KERNEL=="ptmx", SECLABEL{smack}="*"
+KERNEL=="tty", SECLABEL{smack}="*"
 
-SUBSYSTEM=="graphics", GROUP="video", SMACK="*"
-SUBSYSTEM=="drm", GROUP="video", SMACK="*"
-SUBSYSTEM=="dvb", GROUP="video", SMACK="*"
+SUBSYSTEM=="graphics", GROUP="video", SECLABEL{smack}="*"
+SUBSYSTEM=="drm", GROUP="video", SECLABEL{smack}="*"
+SUBSYSTEM=="dvb", GROUP="video", SECLABEL{smack}="*"
 
-SUBSYSTEM=="tty", KERNEL=="ptmx", GROUP="tty", MODE="0666", SMACK="*"
-SUBSYSTEM=="tty", KERNEL=="tty", GROUP="tty", MODE="0666", SMACK="*"
-SUBSYSTEM=="tty", KERNEL=="tty[0-9]*", GROUP="tty", MODE="0620", SMACK="*"
-SUBSYSTEM=="vc", KERNEL=="vcs*|vcsa*", GROUP="tty", SMACK="*"
-KERNEL=="tty[A-Z]*[0-9]|pppox[0-9]*|ircomm[0-9]*|noz[0-9]*|rfcomm[0-9]*", GROUP="dialout", SMACK="*"
+SUBSYSTEM=="tty", KERNEL=="ptmx", GROUP="tty", MODE="0666", SECLABEL{smack}="*"
+SUBSYSTEM=="tty", KERNEL=="tty", GROUP="tty", MODE="0666", SECLABEL{smack}="*"
+SUBSYSTEM=="tty", KERNEL=="tty[0-9]*", GROUP="tty", MODE="0620", SECLABEL{smack}="*"
+SUBSYSTEM=="vc", KERNEL=="vcs*|vcsa*", GROUP="tty", SECLABEL{smack}="*"
+KERNEL=="tty[A-Z]*[0-9]|pppox[0-9]*|ircomm[0-9]*|noz[0-9]*|rfcomm[0-9]*", GROUP="dialout", SECLABEL{smack}="*"
 
-SUBSYSTEM=="input", KERNEL=="mouse*|mice|event*", MODE="0640", SMACK="*"
-SUBSYSTEM=="input", KERNEL=="ts[0-9]*|uinput", MODE="0640", SMACK="*"
+SUBSYSTEM=="input", KERNEL=="mouse*|mice|event*", MODE="0640", SECLABEL{smack}="*"
+SUBSYSTEM=="input", KERNEL=="ts[0-9]*|uinput", MODE="0640", SECLABEL{smack}="*"
-- 
1.8.4.5

